bigquery/deploy/README.md

```diff
@@ -2,17 +2,17 @@
 
 * Set up the Kubernetes environment
   * Create a service account and give it access to the BigQuery tables for your
-  dataset
-    * Navigate to the project where you will deploy Data explorer.
-    * [Following the principle of least privilege](https://cloud.google.com/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform#why_use_service_accounts),
-    we create a service account and give it only the necessary permissions,
-    rather than use the default Compute Engine service account. At the end of
-    this section, the new service account will have access to the BigQuery
-    tables for your dataset. Before creating the service account, you must
-    ensure that everyone who could use this service account already has access
-    to the BigQuery tables. To see who could use this service account, navigate
-    to `IAM & Admin` and look for `Owner`, `Editor`, and `Service Account User`
-    roles.
+  dataset  
+  [Following the principle of least privilege](https://cloud.google.com/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform#why_use_service_accounts),
+  we recommend using a service account with only the necessary permissions,
+  rather than the default Compute Engine service account.
+    * Create a project for deploying Data Explorer. We recommend creating a
```

Comment:
This would be a separate PR, but if we recommend creating a new project, that should address your concern with reserving a static internal IP address for the Elasticsearch client so we can eliminate a bunch of config/steps both here and in deploying data explorer, right?

Reply:
I still don't think we should use static IP for the reasons discussed earlier, let's discuss ofline.

Eliminating the bunch of config/steps can easily be done with a shell script, as I described yesterday. I hope to start this in the next week or so, you're welcome to try too.